x-pack/plugins/observability/public/components/shared/exploratory_view/embeddable/embeddable.tsx

```diff
@@ -19,19 +19,31 @@ import { ReportConfigMap } from '../contexts/exploratory_view_config';
 import { obsvReportConfigMap } from '../obsv_exploratory_view';
 import { ActionTypes, useActions } from './use_actions';
 import { AddToCaseAction } from '../header/add_to_case_action';
+import { ViewMode } from '../../../../../../../../src/plugins/embeddable/common';
+import { observabilityFeatureId } from '../../../../../common';
+import { SingleMetric } from './single_metric';
 
 export interface ExploratoryEmbeddableProps {
-  reportType: ReportViewType;
-  attributes: AllSeries;
+  appId?: 'security' | 'observability';
   appendTitle?: JSX.Element;
+  attributes?: AllSeries;
   title: string | JSX.Element;
   showCalculationMethod?: boolean;
   axisTitlesVisibility?: XYState['axisTitlesVisibilitySettings'];
-  legendIsVisible?: boolean;
+  customHeight?: string | number;
+  customLensAttrs?: any;
+  customTimeRange?: { from: string; to: string };
   dataTypesIndexPatterns?: Partial<Record<AppDataType, string>>;
+  isSingleMetric?: boolean;
+  legendIsVisible?: boolean;
+  metricIcon?: string;
```

Comment:
```suggestion
  metricIcon?: IconType;
```

can probably use IconType from elastic/eui